lib/web/apiserver.go

```diff
@@ -2695,18 +2696,25 @@ func (h *Handler) generateSession(ctx context.Context, clt auth.ClientI, req *Te
 		port = 0
 		id = req.Server
 	}
+	accessChecker, err := scx.GetUserAccessChecker()
```

Comment:
FYI, this method was recently optimized to reduce the latency incurred when starting a session via the web UI.

Do you have a sense of how much extra latency these moderation checks add? Might be worth checking with @rosstimothy who was involved with speeding this up a few months ago.

Reply:
I don't have a particular answer, but `GetUserAccessChecker` doesn't seem to dial anywhere, it only checks from certs. Unless parsing from cert is expensive, I don't believe any meaningful latency would be added. Adding tracing to the web UI would be good but perhaps out of scope for this PR (unless that already exists and I'm unaware of it)

Reply:
Checking through the trace logs for the `GET/sessions` call doesn't seem to add any perceptible latency. There is more variance in latency between repeat calls (within single digits) than there is adding/removing this check. Although I may be looking at the logs wrong. The relevant call imo is `GetCurrentUserRoles` in order to check for policies and that call itself is ~1ms

Reply:
Great, thanks!